### ISSUE TITLE
Lora微调返回代码-7

### ISSUE THREAD

**Issue by HarryK4673** [THUDM/VisualGLM-6B#337]
[2024-01-19 08:39:49,153] [ERROR] [launch.py:321:sigkill_handler] ['/usr/bin/python3', '-u', 'finetune_visualglm.py', '--local_rank=1', '--experiment-name', 'finetune-visualglm-6b', '--model-parallel-size', '1', '--mode', 'finetune', '--train-iters', '300', '--resume-dataloader', '--max_source_length', '64', '--max_target_length', '256', '--lora_rank', '10', '--layer_range', '0', '14', '--pre_seq_len', '4', '--train-data', '/app/dataset.json', '--valid-data', '/app/dataset.json', '--distributed-backend', 'nccl', '--lr-decay-style', 'cosine', '--warmup', '.02', '--checkpoint-activations', '--save-interval', '300', '--eval-interval', '10000', '--save', './checkpoints', '--split', '1', '--eval-iters', '10', '--eval-batch-size', '1', '--zero-stage', '1', '--lr', '0.0001', '--batch-size', '1', '--skip-init', '--fp16', '--gradient-accumulation-steps', '4', '--use_lora'] exits with return code = -7
使用Qlora可以正常运行，但是无法进行推理。如果不输入quant则显示这个变量没有长度，如果指定为4或者8则显示维度不匹配
